src/Datagrid/SimplePager.php

```diff
@@ -22,7 +22,7 @@
 final class SimplePager extends Pager
 {
     /**
-     * @var bool
+     * @var bool|null
```

Comment:
Should we use false as a default value ?

Reply:
Yes, fixed